### PR TITLE
feat(cli): update query by loading it from the platform

### PR DESCRIPTION
**Pull request by afiune**
## User Story

As a Lacework CLI user,
I want to update a query without having to type the query again,
So I can modify my queries faster via inline modifications.

## Summary

This change is adding the ability to load a query into the default
editor from the user so that they can easily make inline modifications.

Simply run the command `lacework query update [query_id]` and an editor
will open with the query preloaded, modify it, save and close the file
for the Lacework CLI to apply those modifications.

Signed-off-by: Salim Afiune Maya <afiune@lacework.net>

## How did you test this change?

Ran it locally to do an inline editing:
lql-inline-update

## Issue

https://lacework.atlassian.net/browse/ALLY-875

### CODE REVIEW

File: integration/lql_update_test.go [hunk 1]
```diff
@@ -27,15 +27,6 @@ import (
...
 
-func TestQueryUpdateHelp(t *testing.T) {
-	out, err, exitcode := LaceworkCLI("help", "query", "update")
-	assert.Contains(t, out.String(), "lacework query update [flags]")
-	assert.Contains(t, out.String(), "-f, --file string")
-	assert.Contains(t, out.String(), "-u, --url string")
-	assert.Empty(t, err.String(), "STDERR should be empty")
-	assert.Equal(t, 0, exitcode, "EXITCODE is not the expected one")
-}
-
```
Comment (afiune) [inline integration/lql_update_test.go:38]: This is already covered globally! Kuddos to @hazedav

**Review by hazedav — APPROVED** [review]

**Review by dmurray-lacework — APPROVED** [review]

**Comment by afiune** [comment]
Modified the pipeline to install vim via:
```
apt-get install -y vim
```

JIRA: https://lacework.atlassian.net/browse/ALLY-924

**Comment by afiune** [comment]
tenor-153374910

**Pull request merged**